federation/pkg/federation-controller/util/meta.go

```diff
@@ -92,3 +98,33 @@ func DeepCopyApiTypeOrPanic(item interface{}) interface{} {
 	}
 	return result
 }
+
+func IsFederationOnlyObject(object runtime.Object) bool {
```

Comment:
Thanks a lot for teaching me this. I was looking for this badly. will adapt the code